### DISCUSSION TITLE
【keyball44】R7C1が反応しない＆しばらくすると左右反転してしまう。

### DISCUSSION THREAD

**Discussion by roppi** [Yowkees/keyball#700] [Q&A]
この度はお世話になります。
ケースの付け替え時に基盤を破損したのか、以下の症状がでるようになってしまいました。
解決に向けてお知恵をお貸しいただけないでしょうか。

【症状】
①R7C1キーが反応しない（OLEDの押下ボタンも更新されない）
②トラックボールが上下左右逆になる
③キー配列が左右反対になる

症状①については常時発生し、②③は条件は判明しませんがそれなりの頻度で発生します。
また、症状②③は個別の事象で、マウスのみ反転する場合とキー配列含めて反転する場合があります。

【症状②③について試したこと】
・TRSケーブルの変更　→　改善せず
・右のProMicroとPCを接続　→　症状発生せず。
　　　　　　　　　　　　　　　左のProMicroに接続して症状②③が発生した場合も、
　　　　　　　　　　　　　　　右につないでから左につなぎなおすと改善する場合がある
・左右のProMicroを交換　→　症状変わらず。（右とPCを接続した場合は症状が発生しない）

【そのほか】
症状①が出た後、以下のことを行いました。
・キーR7C1とR6C1の間の下側のダイオードを交換

よろしくお願いいたします。

![image](https://github.com/user-attachments/assets/ebce905c-e070-49b1-9f94-87e6268d81cf)

**Comment by Yowkees** [comment]
Keyball44をご愛用いただき、誠にありがとうございます。

Keyballを分解・組立を行うと、基板の反り具合が変わり、半田にクラックが入るなどして導通不良になることがあります。
まず右手側のR7C1については、キーの上のダイオード（LED17に一番近いダイオード）が対応しますので、この半田付けを見直してください。すでに部品交換済ということですので、PCBソケットの半田付けもやり直してみてください。

次に左右反転してしまう件ですが、左手側基板のLEFTジャンパーがうまくショートできていない場合に発生します。
LEFTジャンパーを再はんだ付けしてみてください。また、左手側のLED17に一番近いダイオードがジャンパーに対応するダイオードです。この半田付けも見直してみてください。

以上、お手数ですがご確認よろしくお願いいたします。

**Reply by roppi** [reply]
ご回答ありがとうございます。
いただいた内容で再度確認してみます。
